src/components/KTable/KTable.vue

```diff
@@ -106,26 +105,42 @@
                   sortClickHandler(column)
                 }
               }"
+              @mouseleave="currentFocusedColumn = ''"
+              @mouseover="currentFocusedColumn = column.key"
             >
-              <span class="k-table-headers-container">
-                <slot
-                  :column="getGeneric(column)"
-                  :name="getColumnSlotName(column.key)"
-                >
-                  <span :class="{'sr-only': column.hideLabel}">
-                    {{ column.label ? column.label : column.key }}
-                  </span>
-                </slot>
-
-                <KIcon
-                  v-if="!disableSorting && !column.hideLabel && column.sortable"
-                  aria-hidden="true"
-                  class="caret"
-                  :color="`var(--kui-color-text, ${KUI_COLOR_TEXT})`"
-                  icon="chevronDown"
-                  :size="KUI_ICON_SIZE_20"
-                />
-              </span>
+              <div
```

Comment:
this isn't as different as it looks. I had to wrap the inner content of the `<th>` in a new div for the CSS `resize` support, the `<th>` should just be an empty wrapper around this `div` which has all the heading styles applied to it.